app/models/Export.php

```diff
@@ -117,4 +117,18 @@ public static function incomplete_apps_by_first_and_email_query()
 
     return $results;
   }
+
+  public static function yes_applicants_query()
+  {
+    $results = DB::select('SELECT distinct concat(u.first_name, " ", u.last_name) as name, p.gender, p.state, p.zip, group_concat(r.race) as race, REPLACE(REPLACE(REPLACE(REPLACE(a.activities,CHAR(10)," "),CHAR(13)," "), CHAR(9)," "), CHAR(44),";"), concat("footlockerscholarathletes.com/admin/applications/", u.id) as link
```

Comment:
let's add some documentation for those replaces 
and maybe on a new line so it's a little more readable